pages/index.js

```diff
@@ -311,23 +313,23 @@ export default class extends React.Component {
             onHideClick={this.toggleGenerate}
             show={this.state.generated}
             width={[9/10, 7/10]}
-            height={[9/10, 7/10]}
+            // height={[9/10, 7/10]}
```

Comment:
@hellais For some reason, enabling the `height` prop makes the modal use the value as-is. e.g `height={7/10}` assigns `height: '0.699px'`  to the modal content div.

Reply:
Removed `height`